public/js/chat.js

```diff
@@ -39,18 +39,20 @@ socket.on('updateUserList', function(users) {
 
 socket.on('newMessage', function(message) {
   const { from, text } = message;
+  let avatar = message.avatar;
+  if (from === 'Admin') avatar = 'admin-1.png';
```

Comment:
I will need to implement something to avoid that the user could not enter with the name "Admin". Otherwise, it will be a bit confusing if someone will write some bullshit with the name and avatar of the admin of the room...